### ISSUE TITLE
Menopause improves symptoms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10095]
**Preferred term label:**
Menopause improves symptoms

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Breast fibroadenoma
Premenstrual dysphoric disorder

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000061
- label: Ameliorated by menopause
- parents: [Ameliorated by (HP:0025254)](https://hpo.jax.org/app/browse/term/HP:0025254)
- pmid: 
- def: Applies to a sign or symptom that is improved or made more bearable by menopause.
- comment: This term may apply to breast fibroadenoma and premenstrual dysphoric disorder.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10095
- synonyms: Menopause improves symptoms(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO id corrected to HP:0040408
